bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/error/Errors.kt

```diff
@@ -0,0 +1,6 @@
+package jp.co.soramitsu.bootstrap.error
+
+import java.lang.RuntimeException
+
+class AccountException(message:String) : RuntimeException(message)
+class IrohaPublicKeyError(message:String) : RuntimeException(message)
```

Comment:
Why does it end with `Error`? Why not `Exception`?

Reply:
updated. Package and class also renamed